profile/README.md

```
@@ -14,9 +14,9 @@
 
 ## ➡️ How to start with CORE:
 
-> If you are in a competition right now, you got lost. Please go to [coregame.de](https://coregame.de/), sign in, join the competition, create your team & clone the repository that is automatically created for you. This is where you can create your bot.
+> If you are in a competition right now, you got lost. Please go to [coregame.sh](https://coregame.sh/), sign in, join the competition, create your team & clone the repository that is automatically created for you. This is where you can create your bot.
 
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)